### PR TITLE
Bio megatron download fix

### PR DESCRIPTION
**Pull request by ekmb**
1. Branch for Bio-related notebooks changed to `main`, for the rest of the  notebooks to `v1.0.0b3`
2. Fixed Bio-megatron download issue

### CODE REVIEW

File: tutorials/asr/06_Voice_Activiy_Detection.ipynb [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "!pip install unidecode\n",
     "\n",
     "# ## Install NeMo\n",
-    "BRANCH = 'v1.0.0b2'\n",
+    "BRANCH = 'v1.0.0b3'\n",
```
Comment (ekmb) [inline tutorials/asr/06_Voice_Activiy_Detection.ipynb:30]: @fayejf should VAD tutorial be `v1.0.0b3` or `main`?

Comment (fayejf) [reply tutorials/asr/06_Voice_Activiy_Detection.ipynb:30]: Let's use main. If that's not good I will change it to 1.0.0b3. Thanks

File: tools/ctc_segmentation/scripts/cut_audio.py [hunk 2]
```diff
@@ -61,7 +61,9 @@ def add_transcript_to_manifest(
         with open(manifest_updated, 'w', encoding='utf8') as f_updated:
             for i, line in enumerate(f):
                 info = json.loads(line)
-                info['transcript'] = transcripts[i]
+                info['transcript'] = transcripts[i].strip()
+                info['WER'] = round(word_error_rate([transcripts[i]], [info['text']]) * 100, 2)
+                info['CER'] = round(word_error_rate([transcripts[i]], [info['text']], use_cer=True) * 100, 2)
```
Comment (ericharper) [inline tools/ctc_segmentation/scripts/cut_audio.py:66]: Are these changes supposed to be in this PR?

File: tools/ctc_segmentation/scripts/prepare_data.py [hunk 3]
```diff
@@ -140,6 +140,7 @@ def split_text(
         .replace("´", "")
         .replace("--", " -- ")
         .replace("-", " - ")
+        .replace(". . .", "...")
```
Comment (ericharper) [inline tools/ctc_segmentation/scripts/prepare_data.py:143]: Is this change supposed to be in this PR?

**Review by ericharper — COMMENTED** [review]
Thanks for fixing this. My only question is that it looks like a couple of commits were in this PR that weren't related to Bio megatron fix.

File: tools/ctc_segmentation/scripts/prepare_data.py (hunk 3 above)
Comment (ekmb) [reply tools/ctc_segmentation/scripts/prepare_data.py:143]: thanks for catching it, will revert.

**Review by ericharper — APPROVED** [review]
Thanks!

**Pull request merged**